src/Services/Terminal/Terminal.js

```diff
@@ -95,6 +103,9 @@ module.exports = function (settings) {
       .then(processResponse)
       .then(
         (response) => {
+          if (debug) {
```

Comment:
By the way, we have different logging modes. 
Let's use them as described in docs in README.md
So here you need to check for `debug === 2`. And check all console.log statements in this PR.